quickstart/life-of-a-fluentd-event.md

```diff
@@ -224,9 +224,7 @@ Once the events are reported by the [Fluentd](http://fluend.org) engine on the *
 
 ## Learn More
 
-* [Fluentd v0.12 Blog Announcement](http://www.fluentd.org/blog/fluentd-v0.12-is-released)
-* [Fluentd v0.14 Blog Announcement](http://www.fluentd.org/blog/fluentd-v0.14.0-has-been-released)
-* [Fluentd v1.0 Blog Announcement](http://www.fluentd.org/blog/fluentd-v1.0)
+Fluentd is maintained continuously and releases it periodically, follow [official blog announcement](https://www.fluentd.org/blog/tag/announcement) with `announcement` tag.
```

Comment:
Thanks! Good idea!

How about this?

* `releases it` -> `released`
  * Structure: `Fluentd is <maintained ...> and <released ...>`
* Expand the link range
  * The original link range made me feel as if I had to set the tag myself.

```suggestion
Fluentd is maintained continuously and released periodically. Follow [official blog announcement with `announcement` tag](https://www.fluentd.org/blog/tag/announcement).
```

Reply:
Thanks!